core/src/wallet/bitcoin/transaction_builders/BitcoinLikeStrategyUtxoPicker.cpp

```diff
@@ -425,9 +431,21 @@ namespace ledger {
 
             // Minimum amount from which we are willing to create a change for it
             // We take the dust as a reference plus the cost of spending this change
-            const int64_t minimumChange = currency.bitcoinLikeNetworkParameters->DustAmount + signedUTXOCost;
 
-            buddy->logger->debug("Start filterWithKnapsackSolver, target range is {} to {}", amountWithFixedFees, amountWithFixedFees + minimumChange);
+            const int64_t dustAmount_fixedAndOutputPart = BitcoinLikeTransactionApi::computeDustAmount(currency, 
+                                    (fixedSize.Max + (buddy->request.outputs.size() * oneOutputSize)));
+            
+            const int64_t dustAmount_OneInputPart = BitcoinLikeTransactionApi::computeDustAmount(currency, 
```

Comment:
Keep same naming convention for variable.

```suggestion
            const int64_t dustAmountFixedAndOutputPart = BitcoinLikeTransactionApi::computeDustAmount(currency, 
                                    (fixedSize.Max + (buddy->request.outputs.size() * oneOutputSize)));
            
            const int64_t dustAmountOneInputPart = BitcoinLikeTransactionApi::computeDustAmount(currency, 
```